lib/lwt_vmnet.ml

```diff
@@ -90,3 +90,22 @@ let write t c =
     return_unit
   with
   | Vmnet.Error err -> fail (Error err)
+
+let shared_interface_list = Vmnet.shared_interface_list
+
+let add_port_forwarding_rule t protocol ext_port ip int_port =
+  Lwt.catch
+  (fun () ->
+    return (Vmnet.add_port_forwarding_rule t.dev protocol ext_port ip int_port)
+  )(function
+  | Vmnet.Error err -> fail (Error err)
+  | e -> fail e)
+
+let add_tcp_port_forwarding_rule t =
```

Comment:
Very nice! How does one remove a forwarding rule?

Reply:
Ah, I guess you don't.. yet. There are matching calls for listing rules and removing them, but I haven't added bindings in this PR: https://developer.apple.com/documentation/vmnet/vmnet_functions